Sources/SwiftDocC/LinkTargets/LinkDestinationSummary.swift

```diff
@@ -516,5 +574,90 @@ extension LinkDestinationSummary.Variant {
         usr = try container.decodeIfPresent(String?.self, forKey: .usr)
         declarationFragments = try container.decodeIfPresent(LinkDestinationSummary.DeclarationFragments?.self, forKey: .declarationFragments)
         taskGroups = try container.decodeIfPresent([LinkDestinationSummary.TaskGroup]?.self, forKey: .taskGroups)
+        topicImages = try container.decodeIfPresent([TopicImage]?.self, forKey: .topicImages)
+    }
+}
+
+// MARK: - Equatable conformance
```

Comment:
Are these just for use in tests? I'm wondering if we should move them to the test framework if that's the case since they're inherently a little fragile.

Eventually when we support versioning we're going to need to do this across the codebase so we'll want to have a consistent approach for this then.

Reply:
SwiftDocC only uses the equitability in test but it's a public type that already declared itself as equatable so it would be breaking to remove that conformance. 

Also, both SwiftDocCTests and SwiftDocCUtilitiesTests use the this equitability so we would need to add the same implementation in both and keep them up to date which also feels fragile to me.

Reply:
I did try to add the equatable conformance to `RenderReference` and implement manual type erasure with a new `AnyRenderReference` but there were braking API changes anywhere `RenderReference` is used in public API today so I abandoned that idea.

Reply:
Yeah I think that's the approach Maya took here: https://github.com/apple/swift-docc/commit/e4694ab03362bf25fa4ff0ce9596a6aa85add542#diff-76f5ded2a592dfd3700953b0341b1f238d36f3955129c461280b9d502ed509b6. But I think this makes sense for now until we make that more dramatic change.